homeassistant/components/zwave_js/sensor.py

```diff
@@ -230,14 +234,72 @@ def unit_of_measurement(self) -> str | None:
 
         return str(self.info.primary_value.metadata.unit)
 
+
+class ZWaveMeterSensor(ZWaveNumericSensor, RestoreEntity):
+    """Representation of a Z-Wave Meter CC sensor."""
+
+    _attr_state_class = STATE_CLASS_MEASUREMENT
+
+    def __init__(
+        self,
+        config_entry: ConfigEntry,
+        client: ZwaveClient,
+        info: ZwaveDiscoveryInfo,
+    ) -> None:
+        """Initialize a ZWaveNumericSensor entity."""
+        super().__init__(config_entry, client, info)
+
+        # Entity class attributes
+        self._attr_last_reset = dt.utc_from_timestamp(0)
+        self._attr_device_class = DEVICE_CLASS_POWER
+        if self.info.primary_value.metadata.unit == "kWh":
+            self._attr_device_class = DEVICE_CLASS_ENERGY
+
+    async def async_update_last_reset(
```

Comment:
This doesn't need to be a coroutine function since we don't await inside. Make it a `@callback`.
```suggestion
    @callback
    def async_update_last_reset(
```